netbench-orchestrator/src/ssm_utils/netbench_driver.rs

```diff
@@ -4,14 +4,22 @@
 use crate::orchestrator::STATE;
 use std::path::PathBuf;
 
+#[allow(dead_code)]
```

Comment:
why so much dead code?

Reply:
Even though these drivers are available, its possible to use only some of them.

Reply:
`s2n_quic_dc_driver`, `s2n_quic_driver_crates`, `s2n_tls_driver` and `tcp_driver_crates` are being used though, at least I can remove the #[allow(dead_code)] and clippy doesn't complain.

Is the issue that these can only currently be used by changing code, not by configuration?

Reply:
yea these aren't exposed to user configuration.